### PR TITLE
Text corrections from issue #6654 (WIP)

### PR DESCRIPTION
**Pull request by etpalmer63**
Fixes for the typos in the following cards so far:
Collective Restraint
Endless Swarm
Teferi's Curse 
Essence Bottle 
Angel of Renewal
Benediction of Moons
Dwarven Priest

### CODE REVIEW

File: Mage/src/main/java/mage/abilities/effects/common/continuous/GainAbilityAttachedEffect.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
     protected Ability ability;
     protected AttachmentType attachmentType;
     protected boolean independentEffect;
+    protected String targetObjectName;
```
Comment (JayDi85) [inline Mage/src/main/java/mage/abilities/effects/common/continuous/GainAbilityAttachedEffect.java:18]: Missing `targetObjectName` in copy constructor
* `public GainAbilityAttachedEffect(final GainAbilityAttachedEffect effect)`

File: Mage/src/main/java/mage/abilities/effects/common/continuous/GainAbilityAttachedEffect.java [hunk 2]
```diff
@@ -96,7 +102,11 @@ public boolean apply(Game game, Ability source) {
     private void setText() {
         StringBuilder sb = new StringBuilder();
         sb.append(attachmentType.verb());
-        sb.append(" creature ");
+        if (targetObjectName!=null){
+            sb.append(" " + targetObjectName + " ");
```
Comment (JayDi85) [inline Mage/src/main/java/mage/abilities/effects/common/continuous/GainAbilityAttachedEffect.java:106]: 1. Set default `targetObjectName` as `creature` in constructor (so devs can see default values and you don't need null checks).

2. Possible missing changes in `setText`?
* `sb.append(ability.getRule("this creature"));` -- must use `targetObjectName`?

Comment (etpalmer63) [reply Mage/src/main/java/mage/abilities/effects/common/continuous/GainAbilityAttachedEffect.java]: Thanks for the feedback! If I understood correctly, I believe I've made all the suggested changes in commit a6a2511.

File: Mage/src/main/java/mage/abilities/effects/common/continuous/GainAbilityAttachedEffect.java (hunk 1 above)
Comment (etpalmer63) [reply Mage/src/main/java/mage/abilities/effects/common/continuous/GainAbilityAttachedEffect.java:18]: Added in commit a6a2511. Thanks!

**Pull request merged**